commit 1fe3b69536b7da5d6cf6f41819405331c31d3b02
Author: Albert Astals Cid <aacid@kde.org>
Date:   Mon Jun 5 18:08:02 2023 +0200

    Require more modern C++
    
    Otherwise it fails to compile with poppler >= 23.06

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 64dcc559..7ad3507a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,6 +52,10 @@ find_package(KF5 5.69 REQUIRED COMPONENTS
 	XmlGui
 )
 
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
 find_package(Okular5 REQUIRED)
 set_package_properties(Okular5 PROPERTIES
 	DESCRIPTION "Okular core library"
